.github/ISSUE_TEMPLATE/release-team-lead-progress.md

```diff
@@ -0,0 +1,242 @@
+- [ ] Insured that "Removals, Deprecations, and Major Changes"-authors are reviewing the blog before it's being publicized 
+
+**Major Themes & Release Blog:**
+- [ ] Followed up with SIGs on major themes of the release
+- [ ] Checked in with release-notes and release-comms about the status of major themes collection
+
+### 5. Around Code Freeze (~Week 13)
+Defer to the [code freeze section](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/release-team-lead#standards) in the release-team-lead handbook.
+
+#### Shortly before Code freeze
+Code Freeze begins, and it’s now the home stretch of the release. SIGs will need to ensure all work moving forward is carefully curated with required merge labels.
+
+- Monitor the enhancements GitHub project to get an idea of how many PRs are still outstanding leading up to Code Freeze
+- Monitor Testgrid and Prow to understand the stability of the release and PRs getting ready to merge. If Prow and Test grid are not in a good state consult folks from SIG Testing on delaying code freeze by a day if needed.
+- If the release branch is not healthy, stable, and passing tests consistently, notify community through standard channels of need to rectify or code freeze will come early to force focus on stabilization.
+- [ ] Reminded the Branch Manager that branch CI jobs will be needed next week
+- [ ] Send out a reminder email to [kubernetes-dev](https://groups.google.com/a/kubernetes.io/g/dev)
+
+#### With Code Freeze
+- As needed, assist the Bug Triage Lead and Enhancements Lead removing PRs and enhancements from the milestone that aren't merged in time `/milestone clear`
+
+#### After Code Freeze
+- Wait for exception requests and bring them to `#sig-release`
+- [ ] Verified that the [`release-1.XX` branch](https://github.com/kubernetes/kubernetes/branches) has been automatically created at the start of Test Freeze
+- [ ] Verified that the Branch Manager created the CI board on [Testgrid](https://testgrid.k8s.io/sig-release) for the release cycle (1.XX-blocking & 1.XX-informing)
+- [ ] Published the "Removals, Deprecations, and Major Changes blog"
+
+### 6. Test Freeze up to Release Day (~Week 14)
+- [ ] Verified together with the release-docs team that all KEPs with required documentation are ready for review
+- [ ] Completed release theme (slogan, logo and explanation text) and add it to the [release cycle documentation in k/sig-release/releases/release-1.XX](https://github.com/kubernetes/sig-release/tree/master/releases)
+- [ ] Decided with the Release team if burndown meetings are necessary or updates are done via Slack thread on Tuesdays and Thursdays
+- [ ] Discussed Release Lead succession with the EA and sig-release leads
+- [ ] Remind team lead to find successors for the upcoming cycle and discuss candidates with the EA
```

Comment:
non-blocking nit: I would start these conversations a week earlier
```suggestion
- [ ] Discussed Release Lead succession with the EA and sig-release leads
- [ ] Remind team lead to find successors for the upcoming cycle and discuss candidates with the EA
```

Reply:
👍 i will update it tomorrow

Reply:
The succession is mentioned before and this section here is another reminder. So I think it is fine like this.